Create grafana-dashboard.json

lisa opened the pull request:
One may need to customize thresholds for appropriate network values.

README.md update coming later in the day.

fyi @RiRa12621

lisa committed:
Create grafana-dashboard.json

One may need to customize thresholds for appropriate network values.

RiRa12621 commented:
A dropdown to choose the datasource might be helpful, in case not all data sources have the speedtest metric like https://github.com/RiRa12621/speedtest-go-exporter/commit/58148974cb29d4601ee224bf8313db539f2a3d9c but depending on your environment as for defaults.

lisa committed:
Add datasource to grafana template

lisa committed:
Update the README and add sample manifests

This will tie together #4 by providing the Grafana dashboard and making
reference to it. Additionally, sample manifests are provided in case
folks would rather use them over copying/pasting.

Signed-off-by: Lisa Seelye <lisa@users.noreply.github.com>
